src/handlers/completion.rs

```diff
@@ -147,30 +192,90 @@ async fn get_user_matches(
     Ok(result)
 }
 
+fn follow_pipes_for_bucket(call: Box<CallExpr>) -> Option<String> {
+    for arg in call.arguments {
+        if arg.key.name == "bucket" {
```

Comment:
could this have false positives if someone creates a function with a bucket input param as an arg?

Reply:
absolutely, but we don't have a better solution at the moment.